---
title: "DNA methylation, hydroxymethylation and histone modifications"
output: html_document
---

## reviews
* [Cedar, H. and Dor Y. (2019) "Principles of DNA methylation and their implications for biology and medicine." Lancet 392: 777–86](https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(18)31268-6/fulltext)
* [Greenberg, M. V. C. and D. Bourc'his (2019). "The diverse roles of DNA methylation in mammalian development and disease." Nature Reviews: Molecular Cell Biology 20(10): 590-607.](https://www.nature.com/articles/s41580-019-0159-6)
* [Michalak, E. M., et al. (2019). "The roles of DNA, RNA and histone methylation in ageing and cancer." Nature Reviews: Molecular Cell Biology 20(10): 573-589.](https://www.nature.com/articles/s41580-019-0143-1)
* [Jambhekar, A., et al. (2019). "Roles and regulation of histone methylation in animal development." Nature Reviews: Molecular Cell Biology 20(10): 625-641.](https://www.nature.com/articles/s41580-019-0151-1)
* [Lyko, F. (2018). "The DNA methyltransferase family: a versatile toolkit for epigenetic regulation." Nat Rev Genet 19(2): 81-92.](https://www.nature.com/articles/nrg.2017.80)
* [Luo, C., et al. (2018). "Dynamic DNA methylation: In the right place at the right time." Science 361: 1336-1340.](http://science.sciencemag.org/content/361/6409/1336)
* [Wu, X. and Y. Zhang (2017). "TET-mediated active DNA demethylation: mechanism, function and beyond." Nat Rev Genet 18(9): 517-534.](https://www.nature.com/articles/nrg.2017.33)
* [Wu, H. and Y. Zhang (2015). "Charting oxidized methylcytosines at base resolution." Nat Struct Mol Biol 22(9): 656-661.](https://www.nature.com/articles/nsmb.3071)
* [Wu, H. and Y. Zhang (2014). "Reversing DNA methylation: mechanisms, genomics, and biological functions." Cell 156(1-2): 45-68.](http://www.cell.com/cell/fulltext/S0092-8674(13)01588-2)
* [Smith, Z. D. and A. Meissner (2013). "DNA methylation: roles in mammalian development." Nat Rev Genet 14(3): 204-220](https://www.nature.com/articles/nrg3354)  
* [Jones, P. A. (2012). "Functions of DNA methylation: islands, start sites, gene bodies and beyond." Nat Rev Genet 13(7): 484-492](https://www.nature.com/articles/nrg3230)   


## 5hmC genome-wide studies at single-base level
* [Schutsky, E. K., et al. (2018). "Nondestructive, base-resolution sequencing of 5-hydroxymethylcytosine using a DNA deaminase." Nat Biotechnol.](https://www.nature.com/articles/nbt.4204)
* [Wen, L., et al. (2014). "Whole-genome analysis of 5-hydroxymethylcytosine and 5-methylcytosine at base resolution in the human brain." Genome Biol 15(3): R49.](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-3-r49)
* [Yu, M., et al. (2012). "Base-resolution analysis of 5-hydroxymethylcytosine in the mammalian genome." Cell 149(6): 1368-1380.](https://www.sciencedirect.com/science/article/pii/S009286741200534X)  
    TAB-seq method
* [Booth, M. J., et al. (2012). "Quantitative sequencing of 5-methylcytosine and 5-hydroxymethylcytosine at single-base resolution." Science 336(6083): 934-937.](http://science.sciencemag.org/content/336/6083/934.full)  
    oxBS-seq method
    
## case studies
* [Rinaldi, L., et al. (2016). "Dnmt3a and Dnmt3b Associate with Enhancers to Regulate Human Epidermal Stem Cell Homeostasis." Cell stem cell 19(4): 491-501.](https://www.sciencedirect.com/science/article/pii/S1934590916301953?via%3Dihub)

这篇文章证明5hmC除了作为5mC的代谢中间产物，其本身也是有重要功能的biomark

* [Spruijt, C. G., et al. (2013). "Dynamic readers for 5-(hydroxy)methylcytosine and its oxidized derivatives." Cell 152(5): 1146-1159.](https://www.cell.com/abstract/S0092-8674%2813%2900152-9)

质谱的方法鉴定出一批5hmC的reader

* [Iurlaro, M., et al. (2013). "A screen for hydroxymethylcytosine and formylcytosine binding proteins suggests functions in transcription and chromatin regulation." Genome Biol 14(10): R119.](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-10-r119)

promoter序列screening的方法鉴定5hmC和5fC的reader，只选了特定的2个promoter并且是用PCR的方法生产不同种类修饰的序列，和in vivo状态不符
